python/ray/serve/scripts.py

```diff
@@ -454,6 +466,16 @@ def run(
             visualizer = GraphVisualizer()
             visualizer.visualize_with_gradio(handle)
         else:
+            if reload:
+                for changes in watchfiles.watch(
+                    working_dir,
+                    rust_timeout=10000,
+                    yield_on_timeout=True,
+                ):
+                    if changes:
+                        cli_logger.info("Files changed. Redeploying Serve app.")
```

Comment:
```suggestion
                        cli_logger.info(f"Detected file change in path {working_dir}. Redeploying Serve app.")
```
helps make the behavior a little more clear